dcbr-web/src/components/PublicSearch.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="grey lighten-4">
+    <v-content class="mx-4 mb-4">
+      <v-container>
+        <v-layout row wrap>
+          <v-flex mt-2>
+            <h1>Public Search</h1>
+          </v-flex>
+          <v-flex>
+            <v-text-field v-model="searchQuery" label="Last Name or Reg #" />
+          </v-flex>
+          <v-flex mt-2>
+            <v-btn text icon @click="search()">
+              <v-icon color="primary">search</v-icon>
+            </v-btn>
+          </v-flex>
+        </v-layout>
+        <v-layout mb-4>
+          <div>Enter a last name or valid registration number in the searchbar to see the operator's details.</div>
```

Comment:
Question for @mk08sh : should  we explicitly state that the search will only return exact matches?